Assets/MixedRealityToolkit/Providers/BaseSpatialObserver.cs

```diff
@@ -24,18 +24,18 @@ public class BaseSpatialObserver : BaseDataProvider, IMixedRealitySpatialAwarene
             uint priority = DefaultPriority, 
             BaseMixedRealityProfile profile = null) : base(registrar, spatialAwarenessSystem, name, priority, profile)
         {
-            if (MixedRealityToolkit.SpatialAwarenessSystem != null)
-            {
-                SourceId = MixedRealityToolkit.SpatialAwarenessSystem.GenerateNewSourceId();
-            }
-            else
-            {
-                Debug.LogError($"A spatial observer is registered in your service providers profile, but the spatial awareness system is turned off. Please either turn on spatial awareness or remove {name}.");
-            }
+            SpatialAwarenessSystem = spatialAwarenessSystem;
 
+            SourceId = (SpatialAwarenessSystem != null) ? SpatialAwarenessSystem.GenerateNewSourceId() : 0;
```

Comment:
This drops the LogError - was this old error not helpful/we don't want to preserve it? Also, is there a concept of a globally non-valid ID (i.e. is zero just understood to be a non-valid ID)?

Reply:
from what i have seen elsewhere in the code, 0 is essentially the non-valid ID.

and the LogError was not useful in this case